src/lib/interfaces.ts

```diff
@@ -17,4 +17,9 @@ export interface AutoRestSettings {
   debug: boolean;
   runtimeId: string;
   minimumAutoRestVersion: string;
+}
+
+export interface AutoRestCodeGenerationResult {
+  outputFolder: string;
+  generatedFiles: Map<string, string>;
```

Comment:
don't think you meant to use `Map`: it uses `set`/`get`/`has` for element access. I think you wanna have `{ [fileName: string]: string }`

Reply:
I needed that!